### PR TITLE
Changed contributing guide to href to 2.5 instead of 2.2

### PR DESCRIPTION
**Pull request by dogunbound**
Just some things that need to be updated

### CODE REVIEW

File: contribute.php [hunk 1]
```diff
@@ -50,9 +50,9 @@
 <h2 id="contributing"><a class="h2-link" href="#contributing">Contributing</a><a class="back-to-top" href="#top" title="Top of the page"></a></h2> 
 <h3 id="reporting-bugs"><a class="h3-link" href="#reporting-bugs">Reporting bugs</a></h3>
 <ol>
- <li>Make sure it is really a bug and not a misunderstanding of what SFML is supposed to do. Check the <a href="documentation/2.2/">documentation</a> again to be sure.</li>
+ <li>Make sure it is really a bug and not a misunderstanding of what SFML is supposed to do. Check the <a href="documentation/2.5.1/">documentation</a> again to be sure.</li>
```
Comment (eXpl0it3r) [inline contribute.php:53]: Can you use `/latest/` instead?

File: contribute.php [hunk 2]
```diff
@@ -50,9 +50,9 @@
...
  <li>Make sure the bug has not already been reported. Check <a href="https://github.com/SFML/SFML/issues?q=">the GitHub issues</a> and <a href="http://en.sfml-dev.org/forums/index.php?action=search">the forum</a>.</li>
- <li>Use the <a href="https://github.com/SFML/SFML">latest revision</a> from GitHub and check if the bug has not already been fixed. Refer to the <a href="tutorials/2.2/compile-with-cmake.php">official tutorial</a> for information on how to build SFML yourself.</li>
+ <li>Use the <a href="https://github.com/SFML/SFML">latest revision</a> from GitHub and check if the bug has not already been fixed. Refer to the <a href="tutorials/2.5/compile-with-cmake.php">official tutorial</a> for information on how to build SFML yourself.</li>
```
Comment (eXpl0it3r) [inline contribute.php:55]: Can you use `/latest/` instead?

**Review by eXpl0it3r — CHANGES_REQUESTED** [review]
Ohh, thanks for spotting that! With `/latest/` we can prevent future occurrence.

File: contribute.php (hunk 1 above)
Comment (dogunbound) [reply contribute.php]: Done

File: contribute.php (hunk 2 above)
Comment (dogunbound) [reply contribute.php]: Done

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thanks for the fix! 🙂

**Pull request merged**